include/deal.II/fe/fe_values.h

```diff
@@ -313,6 +369,24 @@ namespace FEValuesViews
                                  std::vector<typename ProductType<gradient_type,typename InputVector::value_type>::type> &gradients) const;
 
     /**
+     * Same as above, but using a of vector local degree-of-freedom values.
+     *
+     * The DoF values typically would be obtained in the following way:
```

Comment:
Good question. @tjhei -- would you be ok with just canning this one sentence (or paragraph) into a doxygen macro that we can expand in all of these places? I suspect your objection is that we can't keep all of these places synchronized, but this could be avoided if we put the whole text into a macro.